ontology/yaml/resources/HVAC/entity_types/DOAS.yaml

```diff
@@ -81,6 +81,22 @@ DOAS_CHWSC_EFSS_EFVSC_FDPM3X_PHRU_HWSC_RCO2M_RSPC_SFM_SFSS_SFVSC_SSPC:
   - SFVSC
   - SSPC
 
+DOAS_EFSS_EFVSC_SFSS_SFVSC_CHWSC_HWSC_RSPC_SSPC_HTWHLSTC_FDPM3X:
+  description: "Complex multi zone DOAS with heat recovery wheel."
+  is_canonical: true
+  implements:
+  - DOAS
+  - EFSS
+  - EFVSC
+  - SFSS
+  - SFVSC
+  - CHWSC
+  - HWSC
+  - RSPC
```

Comment:
I am assuming that the exhaust fan is controlling its speed to the `return_air_static_pressure_sensor`. If so, please use `EPC` instead of `RSPC`. Technically, since the air leaving the space can only be exhausted (since this is a DOAS unit, we really don't have "return_air"). I suggest re-naming all "return" fields to use "exhaust" instead.